helm/exporter-kubelets/templates/servicemonitor.yaml

```diff
@@ -17,19 +17,25 @@ spec:
     matchNames:
       - "kube-system"
   endpoints:
-  - interval: 15s
-    {{- if .Values.https }}
-    port: https-metrics
+  {{- if .Values.https }}
+  - port: https-metrics
     scheme: https
+    interval: 15s
     tlsConfig:
-      caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-      # Skip verification until we have resolved why the certificate validation
-      # for the kubelet on API server nodes fail.
       insecureSkipVerify: true
```

Comment:
This is just an example in the bare kube-prometheus manifests, you should in production verify your certs, but I decided to not do so for demo purposes. This should be configurable to the user.